inc/class-embed-privacy.php

```diff
@@ -687,6 +687,29 @@ public function get_single_overlay( $content, $embed_provider, $embed_provider_l
 		if ( $is_empty_provider ) {
 			$providers = $this->get_embeds();
 		}
+		$parsed_url = wp_parse_url( home_url() );
+		$domain = $host = $parsed_url['host'];		
+		
+		// Get the main domain of our current host. notice, that WP can run 
+		// on a subdomain itself (e.g. blog.abc.de). If we want to check for 
+		// subdomains (e.g. mydatahub.abc.de), we have to look for subdomains 
+		// of our domain (abc.de), not for subdomains of the WP domain
+		//  (e.g. something.blog.abc.de).
+		
+		if ((!filter_var($host,FILTER_VALIDATE_IP)) && ($host !== 'localhost')) {
```

Comment:
There is a wide variety of hosts that don’t need to be processed here. We should not add specific non-configurable hosts here.